src/api/chat.test.ts

```diff
@@ -10,7 +10,7 @@ describe('chatWithAI', () => {
     vi.clearAllMocks();
   });
 
-  test('should return list of strings with content returned by sidecar', async () => {
+  it('should return list of strings with content returned by sidecar', async () => {
```

Comment:
The consistent-test-it rule says to use `it` inside `describe` blocks and `test` outside.

Reply:
That's ok. Thanks for the bulk rename :-).